build-system/tasks/e2e/describes-e2e.js

```diff
@@ -22,13 +22,14 @@ const {AmpDriver, AmpdocEnvironment} = require('./amp-driver');
 const {Builder, Capabilities} = require('selenium-webdriver');
 const {clearLastExpectError, getLastExpectError} = require('./expect');
 const {installRepl, uninstallRepl} = require('./repl');
+const {isTravisBuild} = require('../../travis');
 const {PuppeteerController} = require('./puppeteer-controller');
 const {SeleniumWebDriverController} = require('./selenium-webdriver-controller');
 
 /** Should have something in the name, otherwise nothing is shown. */
 const SUB = ' ';
-const TIMEOUT = 20000;
-
+const TEST_TIMEOUT = 20000;
+const SETUP_TIMEOUT = 30000;
```

Comment:
This should be a less than `TEST_TIMEOUT` if it's used inside `beforeEach()`. (See my other comment.)

Also, curious: Why do we need to wait 30s between tests? Ideally, we'd like to run setup just once, and reduce `TEST_TIMEOUT` to something less than 20s. (Not sure if that's feasible in this PR, but it's worth a thought.)